index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a href="#">My Logo</a>
+        </div>
+        <nav>
+            <img src="assets/IconMenu.png" alt="">
+        </nav>
+    </header>
+    <section class="headline">
+        <h2>I'm Ashraf Glad to see you!</h2>
+        <div class="para">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class="connect">
+            <p>LET'S CONNECT</p>
+        </div>
+        <div class="icon">
+            <img src="assets/socialmediaicons.png" alt="social media">
+        </div>
```

Comment:
- [ ] Kindly consider extracting each icon from the Figma template because each icon will have a link to your social media account and the list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag) like this 
```
<ul>
   <li> <a href="#"><img src="path for image"/></a></li>
   <li> <a href="#"><img src="path for image"/></a></li>
</ul>
```